slider/library.py

```diff
@@ -187,7 +189,10 @@ def create_db(cls,
 
                 try:
                     beatmap = Beatmap.parse(data.decode('utf-8-sig'))
-                except ValueError as e:
+                except Exception as e:
+                    if skip_exceptions:
+                        logging.exception(f"Failed to parse {path}")
+                        continue
                     raise ValueError(f'failed to parse {path}') from e
```

Comment:
not sure a `ValueError` is the right exception here, we could have caught any type of exception which we would then reraise as a `ValueError`. Might be best to just reraise a generic `Exception`? Or perhaps a new `BeatmapParseError`?

Reply:
I personally find creating large exception hierarchies to low value. It doesn't really matter why this failed, and we will show the original exception because we are doing a `raise from`. users will just look up "what exception do I need to catch here" and it is nice that it is just a built in.